modules/exploits/multi/http/pimcore_unserialize_rce.rb

```diff
@@ -0,0 +1,264 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => "Pimcore 4.x and 5.x Unserialize RCE",
+      'Description' => %q(
+        This module exploits a PHP unserialize() in Pimcore before 5.7.1 to
+        execute arbitrary code. An authenticated user with "classes" permission
+        could exploit the vulnerability.
+
+        The vulnerability exists in the "ClassController.php" class, where the
+        "bulk-commit" method make it possible to exploit the unserialize function
+        when passing untrusted values in "data" parameter.
+
+        Tested on Pimcore 5.6.6, 5.6.5, 5.6.4, 5.6.3, 5.6.2, 5.6.1, 5.6.0, 5.5.4,
+        5.5.3, 5.5.2, 5.5.1, 5.4.4, 5.4.3, 5.4.2, 5.4.1, 5.4.0 with the Symfony
+        unserialize payload
```

Comment:
Could the tested versions be rewritten in this way?

5.4.0-5.4.4, 5.5.1-5.5.4, 5.6.0-5.6.6